File: fabric/src/main/java/dev/gluton/quickspyglasser/fabric/QuickSpyglasserExpectPlatformImpl.java
package dev.gluton.quickspyglasser.fabric;

import dev.gluton.quickspyglasser.QuickSpyglasserExpectPlatform;
import net.fabricmc.loader.api.FabricLoader;

import java.nio.file.Path;

public class QuickSpyglasserExpectPlatformImpl {
    /**
     * This is our actual method to {@link QuickSpyglasserExpectPlatform#getConfigDirectory()}.
     */
    public static Path getConfigDirectory() {
        return FabricLoader.getInstance().getConfigDir();
    }
}

File: fabric/src/main/kotlin/dev/gluton/quickspyglasser/fabric/QuickSpyglasserExpectPlatformImpl.kt
package dev.gluton.quickspyglasser.fabric

import net.fabricmc.loader.api.FabricLoader
import java.nio.file.Path

object QuickSpyglasserExpectPlatformImpl {
	@JvmStatic
	fun configDirectory(): Path = FabricLoader.getInstance().configDir
}
